README.md

```diff
@@ -63,4 +63,7 @@ will be the only hook that is executed.
 > as environment variables prior to installing.
 
 # License
-Copyright &copy; 2018 by ZEIT, Inc. Released under the [MIT License](LICENSE).
+Copyright &copy; 2017-2020 by ZEIT, Inc.
+Copyright &copy; 2020-2021 by Vercel, Inc.
+
```

Comment:
```suggestion
Copyright &copy; 2021 by Vercel, Inc.

```

Reply:
This isn't legally accurate, though. This library pre-dates 2021.

Reply:
It's the standard for the other repos in our GitHub org 👍